api/src/models/Category.test.ts

```diff
@@ -0,0 +1,19 @@
+import {Category, CategoryRaw} from './Category';
+
+describe("Category", () => {
+    let category: Category;
+    beforeEach(() => {
+        category = new Category("test1");
+    });
+
+    it("should return the directory name", () => {
+        expect(category.dir).toEqual("test1");
+    });
+
+    it("should return the category and jobs", () => {
```

Comment:
Maybe add "...in raw format" to the end of the test description, since this is mainly about testing the raw() method.

Reply:
fixed